### ISSUE TITLE
Custom CSS themes

### ISSUE THREAD

**Issue by fanfare** [twitter/the-algorithm#475]
twitter's 'default' look is starting to look boring... maybe give the option to make the site look more fun? what if there was a grimes theme, or maybe a microsoft encarta type theme..

![https://jollo.org/theme1.jpg](https://jollo.org/theme1.jpg)

![https://i.jollo.org/xoYwhYpb.png](https://i.jollo.org/xoYwhYpb.png)

**Comment by PickleNik** [comment]
this is not part of the recommendations algo

**Comment by NodeMixaholic** [comment]
> this is not part of the recommendations algo

ok, but may I ask who asked? this is genuinely a good feature to add, part of algo or naw.

**Comment by NodeMixaholic** [comment]
also, markdown pls.

**Comment by PickleNik** [comment]
> > this is not part of the recommendations algo
> 
> ok, but may I ask who asked? this is genuinely a good feature to add, part of algo or naw.

This is a repo for recommendation algo, when Twitter opensauces the client you can put an issue there

**Comment by ayobamiseun** [comment]
design related issue

**Comment by Lamby777** [comment]
> > > this is not part of the recommendations algo
> > 
> > 
> > ok, but may I ask who asked? this is genuinely a good feature to add, part of algo or naw.
> 
> This is a repo for recommendation algo, when Twitter opensauces the client you can put an issue there

someday... [inhales copium]

**Comment by PickleNik** [comment]
could've just allowed already opensauce alt-clients instead smh 🥲

**Comment by chinedu360** [comment]
hmmm...elon has to see this

**Comment by leafstrat** [comment]
It's already doable with custom CSS browser addons.

Styles are shared here by creators;
https://userstyles.org/styles/browse/twitter

![173088_after](https://user-images.githubusercontent.com/4014775/229256222-9723c5be-7984-4b43-9940-0444cb6b9f37.png)

**Comment by PickleNik** [comment]
> It's already doable with custom CSS browser addons.
> 
> Styles are shared here by creators; https://userstyles.org/styles/browse/twitter
> 
> ![173088_after](https://user-images.githubusercontent.com/4014775/229256222-9723c5be-7984-4b43-9940-0444cb6b9f37.png)

Yeah but I think the point of original issue is to make your page original in your own style for others rather than style a website in a certain way for yourself. MySpace vibes.

**Comment by centunesima** [comment]
This is MySpace.

**Comment by Lamby777** [comment]
> > It's already doable with custom CSS browser addons.
> > Styles are shared here by creators; https://userstyles.org/styles/browse/twitter
> > ![173088_after](https://user-images.githubusercontent.com/4014775/229256222-9723c5be-7984-4b43-9940-0444cb6b9f37.png)
> 
> Yeah but I think the point of original issue is to make your page original in your own style for others rather than style a website in a certain way for yourself. MySpace vibes.

Bonus points if they allow XSS.

Did I mention that Samy is my hero?

**Comment by Gamiac** [comment]
Tumblr is over there.

**Comment by leafstrat** [comment]
> Yeah but I think the point of original issue is to make your page original in your own style for others rather than style a website in a certain way for yourself. MySpace vibes.

Would be cool. I remember when YouTube let you do that.

**Comment by AleksLi1** [comment]
lgtm

**Comment by jdm-twtr** [comment]
This issue was closed due to possibility of spam. Please re-file with a useful title and description.
